include/fastdds/dds/publisher/DataWriter.hpp

```diff
@@ -349,8 +451,8 @@ class DataWriter : public DomainEntity
      *
      */
     RTPS_DllAPI ReturnCode_t get_matched_subscription_data(
-            SubscriptionBuiltinTopicData& subscription_data,
-            fastrtps::rtps::InstanceHandle_t subscription_handle) const;
+            builtin::SubscriptionBuiltinTopicData& subscription_data,
+            const fastrtps::rtps::InstanceHandle_t subscription_handle) const;
```

Comment:
```suggestion
            const fastrtps::rtps::InstanceHandle_t& subscription_handle) const;
```
When changed, close also Iker's comment above.